voicer-client/src/components/TalentProfile.js

```diff
@@ -1,12 +1,95 @@
 import React from 'react';
 import { Form, FormGroup, Label, Input, Button } from 'reactstrap';
+import Select, { components } from 'react-select';
+import { connect } from 'react-redux';
+import {
+  getLanguages,
+  getAccents,
+  addTalentLanguage,
+  addTalentAccent
+} from '../actions/register';
+import makeAnimated from 'react-select/animated';
+import 'bootstrap/dist/css/bootstrap.css';
+
+const ageOptions = [
+  { value: 'child', label: 'Child' },
+  { value: 'teen', label: 'Teen' },
+  { value: 'adult', label: 'Adult' },
+  { value: 'senior', label: 'Senior' }
+];
+
+// const languageOptions = [
+//     { value: 'english', label: 'English' },
+//     { value: 'spanish', label: 'Spanish' },
+//     { value: 'german', label: 'German' },
+//     { value: 'japanese', label: 'Japanese' },
+//     { value: 'chinese', label: 'Chinese' }
+// ];
+
+// const accentOptions = [
+//     { value: 'american', label: 'American' },
+//     { value: 'british', label: 'British' },
+//     { value: 'central american', label: 'Central American' },
+//     { value: 'european spanish', label: 'European Spanish' }
+// ];
 
 class TalentProfile extends React.Component {
-  state = {
-    gender: '',
-    voiceAge: '',
-    languages: '',
-    biography: ''
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      gender: '',
+      voiceAge: '',
+      languageOptions: [],
+      accentOptions: [],
+      languages: [],
+      accents: [],
+      biography: ''
+    };
+  }
+
+  componentDidMount() {
+    this.props.getLanguages().then(this.modifyLanguage);
+    this.props.getAccents().then(this.modifyAccents);
+  }
+
```

Comment:
might want to add a comment here about how modifyLanguage is working with the Redux store.

Reply:
added Comment explaining CDM process and modifyLanguage